src/docs/ContrastBox.jsx

```diff
@@ -3,15 +3,15 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 const contrastBlockCssClass = 'docs-block__contrast-box';
-const ContrastBox = ({toBottom, smallPadding, light, fullWidth, children}) => {
+const ContrastBox = ({toBottom, smallPadding, light, fullWidth, children, style}) => {
```

Comment:
it's used only in docs, i would stick to this solution, it allows us to adjust this ContrastBox if needed.